SPECS-EXTENDED/rubygem-asciidoctor/rubygem-asciidoctor.spec

```
@@ -1,35 +1,63 @@
 %global debug_package %{nil}
 %global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
 %global gem_name asciidoctor
-
 Summary:        A fast, open source AsciiDoc implementation in Ruby
 Name:           rubygem-%{gem_name}
-Version:        2.0.15
+Version:        2.0.17
 Release:        1%{?dist}
 Group:          Development/Languages
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:	Mariner
-URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
-Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+URL:            https://asciidoctor.org/
+Source0:        https://github.com/%{gem_name}/%{gem_name}/archive/refs/tags/v%{version}.tar.gz#%{gem_name}-%{version}.tar.gz
```

Are we sure this works? Usually there's a `/` needed in front of the file name:

```suggestion
Source0:        https://github.com/%{gem_name}/%{gem_name}/archive/refs/tags/v%{version}.tar.gz#/%{gem_name}-%{version}.tar.gz
```